model/common/src/icon4py/model/common/interpolation/interpolation_attributes.py

```diff
@@ -131,4 +136,44 @@
         icon_var_name="cells_aw_verts",
         dtype=ta.wpfloat,
     ),
+    RBF_VEC_COEFF_C1: dict(
+        standard_name=RBF_VEC_COEFF_C1,
+        long_name="rbf interpolation coefficient from cells to edges 1",  # TODO: name
```

Comment:
I suppose this `long_name` has the same questions as in https://github.com/C2SM/icon4py/pull/709/files#r2111186708.

Reply:
Again, it would be valuable to know if 1 is cell-tangential and 2 is cell-normal, or possibly the other way around.